cost-analyzer/templates/query-service-deployment-template.yaml

```diff
@@ -44,12 +44,15 @@ spec:
       {{- end }}
     spec:
       restartPolicy: Always
-      {{- if .Values.kubecostDeployment.queryService.securityContext }}
+      {{- if and .Values.global.platforms.openshift.enabled .Values.global.platforms.openshift.securityContext }}
       securityContext:
-        {{- toYaml .Values.kubecostDeployment.queryService.securityContext | nindent 8 }}
+        {{- toYaml .Values.global.platforms.openshift.securityContext | nindent 8 }}
       {{- else if .Values.global.securityContext }}
       securityContext:
         {{- toYaml .Values.global.securityContext | nindent 8 }}
+      {{- else if .Values.kubecostDeployment.queryService.securityContext }}
```

Comment:
Same as above

Reply:
Posted comment above.

Reply:
Posted reply and making changes throughout for consistency sake.